lib/util/identifier.js

```diff
@@ -354,3 +355,7 @@ exports.getUndoPath = (filename, outputPath, enforceRelative) => {
 		? `./${append}`
 		: append;
 };
+
+exports.getRealIdentity = str => {
+	return str.replace(/\0#/g, "#");
```

Comment:
```suggestion
	return str.replace(/\0(.)/g, "$1");
```

`\0` can escape anything